package.json

```diff
@@ -140,15 +140,11 @@
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.17.2",
     "svg-inline-loader": "^0.8.2",
-    "typescript": "^3.8.3",
+    "typescript": "^3.9.2",
     "url-loader": "^4.1.0",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3",
+    "webpack-dev-server": "^3.11.0",
     "webpack-merge": "^4.2.2"
-  },
-  "resolutions": {
-    "minimist": "^1.2.5",
-    "webpack-dev-server/yargs": "~13.2"
```

Comment:
@davisagli Note these changes -- if you look at `yarn.lock` below you can see that these are still satisfied.